NextExample/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "analyze": "ANALYZE=true next build",
+    "cache:clear": "rm -rf .expo .next node_modules/.cache"
+  },
+  "dependencies": {
+    "next": "latest",
+    "raf": "^3.4.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-native-reanimated": "3.0.0-rc.3",
```

Comment:
Is there any option to use `link:../` here so we can have fast refresh when we edit package files? cc @nandorojo

Reply:
an easier method is probably to un-comment the `webpack` config resolver in `next.config.js` and point it to the right place. but `link` might also work. i assume this is using `yarn link`? i don’t think i’ve tried this before

Reply:
Does this approach support fast-refresh also when changing Reanimated codebase, e.g. `src/reanimated2/core.ts`?